internal/pkg/agent/application/monitoring/processes.go

```diff
@@ -50,7 +50,15 @@ func processesHandler(coord *coordinator.Coordinator) func(http.ResponseWriter,
 
 		for _, c := range state.Components {
 			if c.Component.InputSpec != nil {
-				procs = append(procs, process{c.Component.ID, c.Component.InputSpec.BinaryName,
+				displayID := c.Component.ID
+				if strings.Contains(displayID, "apm") {
```

Comment:
I think this might do unexpected things if we ever have another APM related input. Going to switch to looking at the actual binary name.